.storybook/preview-head.html

```diff
@@ -1,48 +1,8 @@
 <link
-  rel="preload"
-  href="/static/media/Lato-Black.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-BlackItalic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Bold.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-BoldItalic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Italic.woff2"
-  as="font"
-  type="font/woff2"
-  crossorigin="anonymous"
-/>
-
-<link
-  rel="preload"
-  href="/static/media/Lato-Regular.woff2"
+  rel="preload stylesheet"
```

Comment:
```suggestion
  rel="preload"
```

I don't think `preload stylesheet` is a valid value for the rel attribute here: https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload

Reply:
You need to declare these as separate links.
```html
<link rel="preload" as="style" href="https://fonts.sage.com/sage.css" crossorigin="anonymous" />
<link rel="stylesheet" href="https://fonts.sage.com/sage.css" crossorigin="anonymous" />
```